docs/integrations/webhooks/create-a-webhook.mdx

```diff
@@ -0,0 +1,261 @@
+---
+title: Create a Clerk webhook in your Next.js application
+description: Learn how to create a Clerk webhook with Svix.
+---
+
+# Create a Clerk webhook in your Next.js application
+
+<TutorialHero
+  beforeYouStart={[
+    {
+      title: "Set up a Clerk + Next.js application",
+      link: "https://clerk.com/docs/quickstarts/nextjs"
+    }
+  ]}
+>
+- Listen for an event in the Clerk Dashboard
+- Create a webhook handler that
+  - uses Svix to verify the webhook signature
+  - receives the webhook's payload
+- Test the webhook locally using ngrok and the Clerk Dashboard
+- Add logic to your application to trigger the webhook
+</TutorialHero>
+
+In this guide, you will learn how to create a Clerk webhook in your Next.js application. You will listen for the `user.updated` event by creating a webhook endpoint in the Clerk Dashboard, creating a webhook handler in your Next.js application, and testing the webhook locally using ngrok and the Clerk Dashboard.
+
+This guide can be adapted to listen for any Clerk event.
+
+## Set up ngrok
+
+To test a webhook locally, you will need to expose your local server to the internet. For this guide, you will use [ngrok](https://ngrok.com/).
+
+1. Go to the [ngrok website](https://ngrok.com/) and create an account.
+2. Once you have made it to the [ngrok dashboard](https://dashboard.ngrok.com/get-started/setup/macos), in navigation sidebar, select [**Domains**](https://dashboard.ngrok.com/cloud-edge/domains).
+3. Select **Create domain**.
+4. ngrok will generate a free, non-ephemeral domain for you and a command to start a tunnel with that domain. The command should look something like this:
+
+`ngrok http --domain=fawn-two-nominally.ngrok-free.app 80`
+
+This command will start a tunnel from `https://fawn-one-nominally.ngrok-free.app` to your local server on port 80.
```

Comment:
I would delete this item, and just use 3000 from the jump. Maybe add a 'you can specify a different port if your app runs on another port, or if you wish to use ngrok with another application and service'